compiler-core/src/javascript/tests/snapshots/gleam_core__javascript__tests__bit_arrays__explicit_sized.snap

```diff
@@ -1,11 +1,5 @@
 ---
 source: compiler-core/src/javascript/tests/bit_arrays.rs
-assertion_line: 71
 expression: "\nfn go() {\n  <<256:size(4)>>\n}\n"
 ---
-import { toBitArray, sizedInt } from "../gleam.mjs";
-
-function go() {
-  return toBitArray([sizedInt(256, 4)]);
-}
-
+import { toBitArray } from "../gleam.mjs";
```

Comment:
What happened to these two tests?

Reply:
This snapshot correspond to this test:
https://github.com/gleam-lang/gleam/blob/f863f0ba57c184b6b50eef012bc2a0f39204b160/compiler-core/src/javascript/tests/bit_arrays.rs#L70-L78
Since the size of this array is being set to 4 it would not be byte aligned and it would cause a runtime error. So instead this code marks that as unsupported and so removes the function it is contained in.

I guess I can rewrite these tests to have a size which is a multiple of 8 instead and also add in some new tests which use not byte aligned bit arrays.